cli/azd/pkg/project/framework_service_dotnet.go

```diff
@@ -191,3 +210,36 @@ func normalizeDotNetSecret(key string) string {
 	// ":".
 	return strings.ReplaceAll(key, "__", ":")
 }
+
+// findDotnetProjectFile locates the project file to pass to the `dotnet` tool for a given dotnet service. In the case where
+// [Path] for the the service results in a directory, if current directory contains a single file matching the
+// glob `*.*proj`, then project file will be returned. If there are multiple or no project file, error is returned
+// and user will be asked to add project file path under "project" in azure.yaml.
+func findDotnetProjectFile(serviceConfig *ServiceConfig) (string, error) {
+	info, err := os.Stat(serviceConfig.Path())
+	if err != nil {
+		return "", err
+	}
+
+	if info.IsDir() {
```

Comment:
Consider inverting the if statement to reduce code nesting with a short circuit return of the path.